_data/translation-management-systems.yaml

```diff
@@ -0,0 +1,669 @@
+#      tone:
+#      adaptive:
+#
+#  features:
+#    - name:
+#      description:
+
+- id: crowdin
+  name: crowdin
+  description: Translation management system
+  url: https://www.crowdin.com
+  cat: true
+  documentation: https://support.crowdin.com/configuring-machine-translation-engines/
+  machine_translation_integrations:
+    - service: amazon
+      custom: true
+      glossary: true
+
+    - service: deepl
+      custom: true
+      tone: true
+
+    - service: google
+      custom: true
+      glossary: true
+
+    - service: intento
+      custom: true
+
+    - service: microsoft
+      custom: true
+
+    - service: modernmt
+      custom: true
+
+    - service: watson
+      custom: true
+
+    - service: yandex
+
+  machine_translation_plugins:
+    - service: globalese
+
+  features:
+    - name: Local machine translation engine
+      description: Users can use their local machine translation in the tool
+
+    - name: Suggestions from multiple machine translation engines
+      description: Users can see segment-level suggestions from several machine translation systems at the same time
+
+- id: lilt
+  name: Lilt
+  description: Computer-assisted software with interactive machine translation features
+  url: https://www.lilt.com
+  cat: true
+  documentation: https://support.lilt.com/kb/
+  machine_translation_integrations:
+    - service: lilt
+      custom: true
+      glossary: true
+      adaptive: true
+
+- id: matecat
+  name: MateCat
+  description: Web-based computer-assisted translation tool
+  url: https://www.matecat.com
+  cat: true
+  documentation: https://site.matecat.com/support/
+  machine_translation_integrations:
+    - service: apertium
+
+    - service: google
+      custom: true
+
+    - service: intento
+
+    - service: language-weaver
+
+    - service: microsoft
+      custom: true
+
+    - service: modernmt
+      custom: true
+
+    - service: moses
+
+    - service: prompsit
+
+    - service: smartmate
+
+    - service: tauyou
+
+    - service: tilde
+      custom: true
+
+    - service: yandex
+
+- id: memoq
+  name: MemoQ
+  description: Translation and localization management software
+  url: https://www.memoq.com
+  cat: true
+  documentation: https://docs.memoq.com/current/en/Welcome/welcome-memoq-docs.html
+  machine_translation_integrations:
+    - service: alexa-ai
+
+    - service: amazon
+      custom: true
+      glossary: true
+
+    - service: crosslang
+
+    - service: deepl
+      glossary: true
+      tone: true
+
+    - service: etranslation
+
+    - service: google
+      custom: true
+      glossary: true
+
+    - service: intento
+      glossary: true
+
+    - service: kantanmt
+      custom: true
+
+    - service: microsoft
+      custom: true
+
+    - service: mirai
+
+    - service: pangeamt
+      glossary: true
+
+    - service: systran
+      custom: true
+
+    - service: textra
+
+    - service: tilde
+      custom: true
+      glossary: true
+
+    - service: tmxmall
```

Comment:
Another aggregator. :-)

Reply:
Yeah, there are several aggregators... Should we separate them into a different category or delete them altogether?

Reply:
in some ways they're more like integrations than engines

I'd keep them separate.